pds_doi_core/db_util/doi_database.py

```diff
@@ -0,0 +1,422 @@
+#!/bin/python
+#
+#  Copyright 2020, by the California Institute of Technology.  ALL RIGHTS
+#  RESERVED. United States Government Sponsorship acknowledged. Any commercial
+#  use must be negotiated with the Office of Technology Transfer at the
+#  California Institute of Technology.
+#
+# ------------------------------
+
+import datetime
+import os
+
+import sqlite3
+from sqlite3 import Error
+
+from pds_doi_core.util.general_util import get_logger
+
+# Get the common logger and set the level for this file.
+import logging
+logger = get_logger('pds_doi_core.db_util.doi_database')
+
+class DOIDataBase:
+    # This DOIDataBase class provides mechanism to write a row, update a row into sqlite database.
+    m_database_name = None
+    m_my_conn = None
+    m_NUM_COLS = 13    # We are only expecting 13 colums in the doi table.  If table structure, this value needs updated.
+    m_default_table_name = 'doi' # Default name of table.
+    m_default_db_file    = 'doi.db' # Default name of database.
```

Comment:
this can be in the configuration file.

Reply:
These 2 variables are now retrieved from the conf.ini.default file.

Reply:
ok, thanks